### ISSUE TITLE
Spoken output of TalkBack gets sometimes truncated

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#174]
```
What steps will reproduce the problem?
1. On Galaxy Note 2, start TalkBack
2. Use your device
3. Sooner or later, some spoken output gets truncated regardless of speech 
engine used

What is the expected output? What do you see instead?
I expect all the spoken output to be spoken fully

What version of the product are you using? On what operating system?
TalkBack 3.2.1 on Samsung Galaxy Note 2, Android 4.1.1, Czech locale

Please provide any additional information below.


```

Original issue reported on code.google.com by `tvalu...@gmail.com` on 7 Feb 2013 at 8:03

**Comment by GoogleCodeExporter** [comment]
```
Do you have the "Use proximity sensor" feature enabled in TalkBack settings?  
We've seen in some cases that users accidentally trigger the proximity sensor 
by unintentionally waving a hand over the device, thus silencing speech.
```

Original comment by `caseybur...@google.com` on 12 Feb 2013 at 3:27

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed with the new queueing mechanism used in TalkBack 3.3.0 Beta 1.
```

Original comment by `caseybur...@google.com` on 16 Feb 2013 at 3:43
- Changed state: **Fixed**
